app/components/homeScreen/DailyReads.tsx

```diff
@@ -228,16 +229,18 @@ const DailyReads = ():any => {
       }
       categoryActivityData = categoryActivityData.filter((i: any) => !gamesarray.find((f: any) => f === i.id));
       const activityDataToShow = categoryActivityData[Math.floor(Math.random() * categoryActivityData.length)];
-      const data: any = [];
+      const articleDailyReadsData: any = [];
+      const activityDailyReadsData: any = [];

```

Comment:
These two names are very similar which makes hard to read. Can we rename them to 
dailyArticles and dailyActivityList